src/collections/_documentation/workflow/releases.md

```diff
@@ -29,20 +30,16 @@ Include a release ID (a.k.a version) where you configure your client SDK. This i
 
 How you make the version available to your code is up to you. For example, you could use an environment variable that is set during the build process.
 
-This annotates each error with the release. We recommend that you tell Sentry about a new release prior to deploying it as this will unlock a few more features (explained in Step 2) . But if you don’t, Sentry will automatically create a release entity in the system the first time it sees an error with that release ID.
+This tags each event with the release value. We recommend that you tell Sentry about a new release prior to deploying it as this will unlock a few more features (explained in Step 2) . But if you don’t, Sentry will automatically create a release entity in the system the first time it sees an event with that release ID.
```

Comment:
```suggestion
This tags each event with the release value. We recommend that you tell Sentry about a new release before deploying it, as this will unlock a few more features (explained in Step 2). But if you don’t, Sentry will automatically create a release entity in the system the first time it sees an event with that release ID.
```